content/en/docs/reference/glossary/kube-scheduler.md

```diff
@@ -4,15 +4,15 @@ id: kube-scheduler
 date: 2018-04-12
 full_link: /docs/reference/command-line-tools-reference/kube-scheduler/
 short_description: >
-  Control plane component that watches for newly created pods with no assigned node, and selects a node for them to run on.
+  Control plane component that watches for newly created Pods that have not yet been assigned to any nodes, and selects a node for them to run on.
```

Comment:
Hi, thanks for raising this PR. I'd suggest you go through the [Documentation Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/) to understand the style for writing Kubernetes docs.